constructor/shar.py

```diff
@@ -74,10 +74,12 @@ def get_header(conda_exec, tarball, info):
             variables["direct_execute_%s" % key] = has_shebang(info[key])
     variables["initialize_conda"] = info.get("initialize_conda", True)
     variables["initialize_by_default"] = info.get("initialize_by_default", False)
+    variables["add_condabin_to_path"] = info.get("add_condabin_to_path", True)
```

Comment:
This contradicts the documented default value for `add_condabin_to_path`

Reply:
```suggestion
    variables["add_condabin_to_path"] = info.get("add_condabin_to_path", False)
```